### PR TITLE
Pin Consul and Vault to the latest MPL releases

### PR DESCRIPTION
**Pull request by priteau**

### CODE REVIEW

**Review by bbezak — APPROVED** [review]

File: etc/kayobe/inventory/group_vars/all/vault [hunk 1]
```diff
@@ -10,13 +10,13 @@ seed_hashicorp_registry_password: "{{ stackhpc_docker_registry_password if stack
 seed_consul_docker_image: "{{ stackhpc_docker_registry ~ '/' if stackhpc_sync_hashicorp_images | bool else '' }}hashicorp/consul"
 
 # Seed Consul container image tag.
-seed_consul_docker_tag: "1.16"
+seed_consul_docker_tag: "1.16.1"
```
Comment (markgoddard) [inline etc/kayobe/inventory/group_vars/all/vault:13]: Do we know what will be the first BSL release? If it's 1.17 we could stick with 1.16 here.

Comment (bbezak) [reply etc/kayobe/inventory/group_vars/all/vault:13]: we know other way around - which are the last MPL ones:

https://www.hashicorp.com/license-faq#products-covered-by-bsl

Comment (bbezak) [reply etc/kayobe/inventory/group_vars/all/vault:13]: therefore we're assuming that 1.16.2 will be BSL rls, so pinning to 1.16.1

Comment (priteau) [reply etc/kayobe/inventory/group_vars/all/vault:13]: It's not clear which version will be the first BSL. We did talk about pinning to patch versions anyway, to ensure all our deployments are using the same versions.

Comment (markgoddard) [reply etc/kayobe/inventory/group_vars/all/vault:13]: I expect they'll bump the minor for BSL so that they can release MPL fixes on patch versions.

Comment (priteau) [reply etc/kayobe/inventory/group_vars/all/vault:13]: We could still merge this to get exactly the same container, like we did for Pulp in 3e60f9bd4bcf799a9824b85c3149a024b487629b?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**